src/components/formWrapper/formWrapperTabs.js

```diff
@@ -1,16 +1,27 @@
 import CrossForm from '@/components/crossForm/CrossForm'
 import NFTWrapper from '@/components/nftForm/NFTWrapper'
+import NFTWrapperCommingSoon from '@/components/nftForm/NFTWrapperCommingSoon'
+
+const nftAvailable = process.env.VUE_APP_NFT_AVAILABLE == 'true'
 
 const formWrapperTabs = [
   {
     label: 'Token',
     component: CrossForm,
   },
-  {
+]
+
+if (nftAvailable) {
+  formWrapperTabs.push({
     label: 'NFT',
     component: NFTWrapper,
-  },
-]
+  })
+} else {
+  formWrapperTabs.push({
+    label: 'NFT - Soon',
```

Comment:
Coming soon? 😛

Reply:
You can leave the label as NFT and if they click they will see the big message from coming soon

Reply:
Done at https://github.com/rsksmart/tokenbridge-ui/pull/47/commits/c3975a86bce75432c7c7b407bccc0ed05b6a49a5